### DISCUSSION TITLE
How to disable the Table Wizard when pasting with CTRL+C?

### DISCUSSION THREAD

**Discussion by alambicbicephale** [texstudio-org/texstudio#3282] [Q&A]
It's all in the title. Depending on what I'm pasting TexStudio decides to open the table wizard rather than simply pasting the text. I've looked in the options but can't find how to disable this new feature (very practical in other cases). Is this even possible? If not I think it should. Using v. 4.6.3 qith Qt 6.5.2 on Windows 10.

**Comment by sunderme** [comment]
it opens the table wizard when you paste contents of spreadsheet.

**Reply by alambicbicephale** [reply]
Thanks! In my case, the table wizard opens when I paste bulleted lists copied from LibreOffice Writer... So it would be much simpler just to be able to deactivate it.

**Reply by octaeder** [reply]
@sunderme when I paste from Win 10 MS Excel the cell content is simply pasted with current build. Is this intended?

**Comment by sunderme** [comment]
the detection that a table from calc is pasted has been improved.
